.github/workflows/test-digital-ocean-1-click.yaml

```diff
@@ -59,24 +60,11 @@ jobs:
         done
         echo "All pods are now ready!"
 
-    - name: Wait until all the resources are fully deployed in k8s
-      uses: jupyterhub/action-k8s-await-workloads@main
-      with:
-        namespace: "posthog"
-        timeout: 300
-        max-restarts: 10
-
     # TODO
     # - setup ingestion test
     # - run ingestion test
     # - run k8s spec test
 
-    - name: Emit k8s namespace report
```

Comment:
I didn't find this reporting useful. I think it can be superseded by  k8s spec test